CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ unreleased
 - Enable Google Pay support
 - Fix issue where non utf-8 encoded sites would show strange characters for card placeholders
 - Fix issue where card fields could not be focused by clicking on the corresponding label
+- Allow card form to not be cleared after succesful tokenization with `card.persistCardDataAfterTokenization`
```

Comment:
How do we feel about the naming for this? Too verbose? Do we like the word `tokenization` in here? We don't use it in the Drop-in docs, even though that is what is happening.

Reply:
Maybe something to the effect of 'allowEditingOfSavedPaymentMethods()' it's just as verbose but might be easier to understand.